autogen/main/versions.tf.tmpl

```diff
@@ -24,7 +24,7 @@ terraform {
   required_providers {
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = ">= 3.79.0, <4.0.0"
```

Comment:
I believe we can constrain this to 3.90 based on https://github.com/hashicorp/terraform-provider-google/blob/master/CHANGELOG.md#3900-october-26-2021